tests/test_LDpred2/scripts/ld.sh

```diff
@@ -40,7 +40,7 @@ testSuccess $LDP --ldpred-mode inf --out $fileOut.inf --chr2use 20 21 22
 echo "Test LD estimation with SNP filtering"
 cut -f 1 -d , $fileInputSumStats > $DIR_TESTS/data/snps-for-ld.txt
 testSuccess $LDE --sumstats $DIR_TESTS/data/snps-for-ld.txt rsid --file-ld-chr $dirOut/ld-chr@.rds
-testSuccess $LDP --ldpred-mode auto --out $fileOut.inf --chr2use 20 21 22
```

Comment:
@deepchocolate; does this change look correct? With updates to R (4.0.5->4.4.1) and misc package updates, the tests failed at this point.